test/e2e/tools/eks-anywhere-test-tool/cmd/e2e.go

```diff
@@ -0,0 +1,15 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var e2eCmd = &cobra.Command{
+	Use:   "e2e",
```

Comment:
Looking at this, I wonder if https://github.com/aws/eks-anywhere/blob/main/cmd/integration_test/cmd/test.go#L8 should be integrated with this one and make it all one single `e2e` binary

Which would probably make a lot of sense since the way we fetch the artifacts (this binary) is pretty much tied to the way we run the tests (the other binary in `cmd/integration_test`)

Reply:
I think that'd work well, and eliminate a bit of complexity in maintaining a separate makefile etc